src/Apps/Partner/Routes/Works/index.tsx

```diff
@@ -9,25 +9,30 @@ import {
 } from "Components/ArtworkFilter/ArtworkFilterContext"
 import { BaseArtworkFilter } from "Components/ArtworkFilter"
 import { updateUrl } from "Components/ArtworkFilter/Utils/urlBuilder"
-import { Match, RouterState, withRouter } from "found"
 import { ActiveFilterPills } from "Components/SavedSearchAlert/Components/ActiveFilterPills"
+import { useRouter } from "System/Router/useRouter"
 
 interface PartnerArtworkFilterProps {
   partner: Works_partner
   relay: RelayRefetchProp
-  match?: Match
 }
 
 export const Artworks: React.FC<PartnerArtworkFilterProps> = ({
   partner,
   relay,
-  match,
 }) => {
   const { sidebar } = partner
+  const { match } = useRouter()
+  const filters = match?.location?.query ?? {}
+
+  // Preselects "Recently added" sort option for artsy-2 partner by default
+  if (match?.params?.partnerId === "artsy-2" && filters.sort === undefined) {
```

Comment:
**question / non-blocking**: Would it be worth pulling this slug into a config variable? I remember discussing whether we can reclaim the `artsy` partner slug, at which point we would need to update any hardcoded values.

Maybe something like `ARTSY_MERCHANDISING_PARTNER_SLUGS=artsy-2` and we can later switch to include future slugs which should gain the same behavior: `ARTSY_MERCHANDISING_PARTNER_SLUGS=artsy-2,artsy,artsy-3`

Reply:
yep, sure. I will add the necessary changes

Reply:
One update: the [Artsy MSO account](https://www.artsy.net/partner/artsy) now has the `artsy` slug. `artsy-2` and `artsy-mso` will continue to work as previous slugs but the primary is now `/partner/artsy`.

Reply:
should we add `artsy-2`, `artsy-mso` to `ARTSY_MERCHANDISING_PARTNER_SLUGS`?

Reply:
Sure, let's go ahead and add those too since they don't redirect.

Reply:
Done